File: app/src/main/java/org/michaelbel/application/rest/model/Person.java
package org.michaelbel.application.rest.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

import io.realm.RealmObject;
import io.realm.annotations.Ignore;

@SuppressWarnings("all")
public class Person extends RealmObject implements Serializable {

    @SerializedName("birthday")
    public String birthday;

    @SerializedName("deathday")
    public String deathday;

    @SerializedName("id")
    public int id;

    @SerializedName("name")
    public String name;

    @Ignore
    @SerializedName("also_known_as")
    public List<String> names;

    @SerializedName("gender")
    public int gender;

    @SerializedName("biography")
    public String bio;

    @SerializedName("popularity")
    public double popularity;

    @SerializedName("place_of_birth")
    public String birthPlace;

    @SerializedName("profile_path")
    public String profilePath;

    @SerializedName("adult")
    public boolean adult;

    @SerializedName("imdb_id")
    public String imdbId;

    @SerializedName("homepage")
    public String homepage;
}

File: moviemade/src/main/java/org/michaelbel/moviemade/app/data/model/Person.kt
package org.michaelbel.moviemade.app.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Person(
    @SerialName("birthday") val birthDay: String,
    @SerialName("known_for_department") val knownForDepartment: String,
    @SerialName("deathday") val deathDay: String,
    @SerialName("id") val id: Int,
    @SerialName("title") val name: String,
    @SerialName("also_known_as") val also_known_as: List<String>,
    @SerialName("gender") val gender: Int,
    @SerialName("biography") val biography: String,
    @SerialName("popularity") val popularity: Float,
    @SerialName("place_of_birth") val placeOfBirth: String,
    @SerialName("profile_path") val profilePath: String,
    @SerialName("adult") val adult: Boolean?,
    @SerialName("imdb_id") val imdb_id: String,
    @SerialName("homepage") val homepage: String,
    @SerialName("known_for") val knownFor: List<Movie>
)